Fitness/Views/Cells/PersonalLinkCollectionViewCell.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  PersonalLinkCollectionViewCell.swift
+//  Fitness
+//
+//  Created by Cameron Hamidi on 3/2/19.
+//  Copyright © 2019 Keivan Shahida. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class PersonalLinkCollectionViewCell: UICollectionViewCell {
+    static let identifier = Identifiers.personalSiteCell
+    var siteImageView: UIImageView!
+    var site: PersonalLinkObject!
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        siteImageView = UIImageView()
+        siteImageView.contentMode = .scaleAspectFill
+        contentView.addSubview(siteImageView)
+        
+        setupConstraints()
+    }
+    
+    func configure(for site: PersonalLinkObject) {
+        self.site = site
```

Comment:
Does the cell need to hold a reference to the site? Once the cell is configured for it we probably don't need the model anymore

Reply:
You're right, it doesn't need one; will remove the variable